src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp

```diff
@@ -5944,6 +5944,28 @@ GUI_TEST_CLASS_DEFINITION(test_6816) {
     CHECK_SET_ERR(warningLabel->text().contains("Unable to calculate primer statistics."), "Incorrect warning message");
 }
 
+GUI_TEST_CLASS_DEFINITION(test_6847) {
+    //    1. Open 'human_T1.fa'
+    GTFileDialog::openFile(os, dataDir + "/samples/FASTA", "human_T1.fa");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
```

Comment:
Use GTUtilsSequenceView::checkSequenceViewWindowIsActive(os) instead of waitTaskFinished